commcare/src/main/java/org/motechproject/commcare/domain/report/constants/ColumnType.java

```diff
@@ -0,0 +1,42 @@
+package org.motechproject.commcare.domain.report.constants;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+
+import java.lang.reflect.Type;
+
+public enum ColumnType {
+    FIELD("field"),
+    EXPANDED("expanded");
+
+    private final String type;
+
+    ColumnType(String type) {
+        this.type = type;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public static ColumnType getColumnTypeFromTypeValue(String typeValue) {
```

Comment:
As far as I am understand, the reason why you do not use Enums.valueOf is case-sensitive. However, I believe that it should exist method (maybe in external package) to retrieve enum constant